device/isis/prefixsegmentid.yaml

```diff
@@ -0,0 +1,89 @@
+components:
+  schemas:
+    IsisSR.PrefixSID:
+      description: >-
+        This contains the properties of IS-IS Prefix-SID and its attributes for 
+        the extended Ipv4 and Ipv6 reachability.
+        Refernce: https://datatracker.ietf.org/doc/html/rfc8667#name-prefix-segment-identifier-p.
+      type: object
+      properties:
+        choice:
+          description: |-
+            The choice of Prefix-SID carries a value instead of an index.
```

Comment:
Choice of whether Prefix-SID carries and absolute value or a relative index  to the SRGB/SRLB Ranges.